aws-dual-stack/openshift-cluster.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -euEo pipefail
+scriptName=$(basename "$0")
+workdir=_install_
+##################### EDIT PARAMETERS ###########################
+
+installer_tar="openshift-install-mac.tar.gz"
+client_tar="openshift-client-mac.tar.gz"
+base_domain="DOMAIN_NAME_IN_ROUTE_53"
+ssh_pub_key=$(cat $workdir/openshift-test-ssh.pub)
+pull_secret=$(cat $workdir/pull-secret.txt)
```

Comment:
```suggestion
installer_tar="${:-openshift-install-mac.tar.gz}"
```

should we have `installer_tar=mytarfile.tar ./openshift-cluster.sh`

Reply:
I feel like this just adds more complexity with low value